data/campaigns/Wings_of_Victory/scenarios/01_The_Hunt.cfg

```diff
@@ -0,0 +1,450 @@
+            [/filter]
+            [filter_location]
+                terrain=*^F*,Gg
+            [/filter_location]
+            [filter_location_wander]
+                terrain=*^F*,Gg
+            [/filter_location_wander]
+            [avoid_unit]
+                [not]
+                    type=Rabbit
+                [/not]
+            [/avoid_unit]
+        [/micro_ai]
+        # Rabbit burrows
+        {PLACE_IMAGE scenery/burrow.png 25 37}
+        {PLACE_IMAGE scenery/burrow.png 34 33}
+        {PLACE_IMAGE scenery/burrow.png 40 23}
+        {PLACE_IMAGE scenery/burrow.png 41 31}
+        {PLACE_IMAGE scenery/burrow.png 44 38}
+        {PLACE_IMAGE scenery/burrow.png 49 28}
+        {PLACE_IMAGE scenery/burrow.png 50 44}
+        {PLACE_IMAGE scenery/burrow.png 53 11}
+        # Boars & Rabbits
+        {SCATTER_UNITS 9 "Tusklet,Tusklet,Tusker,Tusker,Tusklet,Tusker" 1 (
+            {EVERYWHERE}
+            terrain=*^F*
+            {SCATTER_NOT_NEXT}
+        ) (side=4)}
+        {SCATTER_UNITS 5 "Rabbit" 1 (
+            {EVERYWHERE}
+            terrain=*^F*
+            {SCATTER_NOT_NEXT}
+        ) (side=4)}
+        [micro_ai]
+            side=4
+            ai_type=forest_animals
+            action=add
+            rabbit_type=Rabbit
+            tusker_type=Tusker,Gorer
+            tusklet_type=Tusklet
+            [filter_location]
+                terrain=*^F*,Gg
+            [/filter_location]
+            rabbit_number=5
+            rabbit_enemy_distance=3
+            ## rabbit_hole_img=scenery/burrow.png # commented out due to bug in forest_animals mai, can't use any other [item] unless I'm fine with rabbits coming out of them until it is fixed.
+            ## Can remove comments and activate for 1.15.0, leave in 1.14.x, as it was only fixed in 1.14.6
+        [/micro_ai]
+
+        {MODIFY_UNIT side=1 facing ne}
+        [objectives]
+            [objective]
+                description= _ "All enemies are defeated and Galun has reached level two."
+                condition=win
+            [/objective]
+            {OBJECTIVE_OR}
+            [objective]
+                description= _ "Turns run out and Galun has reached level two."
+                condition=win
+            [/objective]
+            [objective]
+                description= _ "Death of Galun"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Vank"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "You lose more than one member of your hunting party."
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Turns run out and Galun has not reached level two."
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "All enemies are defeated and Galun has not reached level two."
```

Comment:
This feels like an unfair loss condition. You defeat all your enemies, and then lose. It would make more sense to me to just spawn more enemies if Galun is not yet level 2... though I guess that might undermine the premise a little. Still... why does he absolutely need to be level 2 anyway?

Reply:
It's part of the backstory of the drakes. Short version is that they are not fully mature until they are level 2.

As for the mechanics, it is difficult to defeat all the enemies and not make the objective, and even moreso since the addition of rabbits as they respawn automatically.
Maybe I could revise that somehow...

Reply:
I playtested S1 on 0.9.5 a few days ago and advanced Galun on turn 12 without too much trouble (I actually finished the scenario with Galun being lvl2 with 43/64XP). That objective makes for varied gameplay, has a lore justification, and isn't unreasonably difficult...I don't see a problem with keeping it.